### ISSUE TITLE
Bring back string input tests

### ISSUE THREAD

**Issue by fsoubelet** [pylhc/tfs#32]
I moved tests to using `pathlib.Path` inputs in #30, which in hindsight should have been an addition and not a change.

Let's bring `string` inputs back to tests, in order to test both.
This should be a quite small change: files similar to actual tests, but with the old fixtures inputs using `os.path.join` shenanigans.

**Comment by fsoubelet** [comment]
@JoschD @lmalina 

This brings back `str` inputs in tests (and fixes these nasty typos you spotted in the last PR).
Let me know if you spot more to be done for issue #32.

**Comment by fsoubelet** [comment]
@lmalina 

Commit bf4b0d7 merges the files as discussed.
Tests have names appended with the input type they test (`test_..._pathlib_input` / `test_..._str_input`), as do the pytest fixtures.

Cf commit message, but we have a test not being called in `tests/test_handler` since it doesn't start with `test` (namely, `tfs_indx`). I've checked the repo history and this has been the case since its introduction in commit 26d3599.

Making it callable will have it fail.
I've decided not to change it for now but it might be worth it to see why.
